decaying_enemies/decaying_enemies/Assets/Scriipts/Movement.cs

```diff
@@ -0,0 +1,25 @@
+using UnityEngine;
+
+public class Movement : MonoBehaviour
+{
+    public float velocidad = 3f;
+    public float velocidad2 = 7f;
```

Comment:
change the access modifier as [Serializefield] private and change the names of the variables to english like _speed